test/input.txt

```diff
@@ -148,6 +148,19 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+  # insert in reverse order
+  add Crazy Cook p/999 e/mwahaha@gordonramsay.com a/Not in SG
+  add Billy p/44581269 e/billy@me.com a/somewhere out there
+  add Ah Bob p/97845687 e/ahbob@gmail.com a/14 Toa Payoh Lorong 4 #11-2254
+  sort
+
+  # make sure list indices work after sorting
+  delete 1
```

Comment:
Alternatively, list could also ensure that your list is sorted properly, but your method is fine too.